src/format.py

```diff
@@ -45,13 +45,16 @@ class LineMatch(object):
     ARROW_DECORATOR = '|===>'
 
     def __init__(self, formattedLine, result, index, validateFileExists=False):
+        self.controller = None
+
         self.formattedLine = formattedLine
         self.index = index
 
         (file, num, matches) = result
 
         self.originalFile = file
-        self.file = parse.prependDir(file, withFileInspection=validateFileExists)
+        self.file = parse.prependDir(file,
+                                     withFileInspection=validateFileExists)
```

Comment:
My version of pep8 complained and broke the line with `/`, which I didn't like so I used a new line instead. What version do you have installed?